languages/go/reference/exercise-concepts/two-fer.md

```diff
@@ -0,0 +1,54 @@
+# Two fer
+
+## Exercise
+
+Given a name, return a string with the message: "One for X, one for me.", where the X can be provided and defaults to "you" if not given.
+
+## Concepts encountered while solving it
+
+### Code formatting
+
+It's the first time most students work with go, and many have not yet used `gofmt`.
+
+### Linting
+
+It's the first time most students work with go, and many have not yet used `golint`. Linting already provides some hints
+on what (not) to do, eg use of variable names, not using else if the the regular condition ends with return, and missing
```

Comment:
This isn't quite clear to me (what does 'use of variable names' mean? You should use them? You _shouldn't_ use them?)